src/OpenFeature/Telemetry/EvaluationEvent.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+
+namespace OpenFeature.Telemetry;
+
+/// <summary>
+/// Represents an evaluation event for feature flags.
+/// </summary>
+public class EvaluationEvent
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EvaluationEvent"/> class.
+    /// </summary>
+    /// <param name="name">The name of the event.</param>
+    /// <param name="attributes">The attributes of the event.</param>
+    /// <param name="body">The body of the event.</param>
+    public EvaluationEvent(string name, Dictionary<string, object?> attributes, Dictionary<string, object> body)
+    {
+        this.Name = name;
+        this.Attributes = attributes;
+        this.Body = body;
```

Comment:
Body is no longer used.

Reply:
Removed: https://github.com/open-feature/dotnet-sdk/pull/397/commits/1b984c6ada975849d994628c587c2298cc1fb005